CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 ## 0.5.0
 
 * @iov/crypto: Rename all `Slip0010*` symbols to `Slip10*`
+* @iov/crypto: Fix keypair representation of Secp256k1.makeKeypair
+
+Breaking changes
+
+* Due to multi curve support in keyring entries, UserProfiles stored with
```

Comment:
Every breaking change breaks all keys :)
Good enough, we just need to be careful when we have actual users...